pkg/controller/networkaddonsconfig/pod_controller.go

```diff
@@ -27,6 +27,7 @@ type ReconcilePods struct {
 	resources     []types.NamespacedName
 }
 
+// SetResources updates context's resources
```

Comment:
What warning? Could you elaborate in the commit message?

Reply:
updated cm

"Warning was
pod_controller.go:30: exported method ReconcilePods.SetResources should have comment or be unexported"

Reply:
You mean warning in your IDE or on CI? I'm fine with this change, but in case this is just to silence your local setup, don't take it as a precedent.

Reply:
my ide runs golint which check global styling
all the funcs there in that files respect this style guide as well

Doc Comments
All top-level, exported names should have doc comments, as should non-trivial unexported type or function declarations. See https://golang.org/doc/effective_go.html#commentary for more information about commentary conventions.